# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Alan Kocay, 2015
# bal4nce, 2015
# cyberfork <benni@include-benni.de>, 2013
# cyberfork <benni@include-benni.de>, 2012
# Berkan <thetoerk@gmail.com>, 2013
# Christian Fröhlich <rutz90@gmx.de>, 2013
# Christian Kaindl <crisscross.kaindl@outlook.de>, 2015
# Christoph Engler <minicheffe@googlemail.com>, 2015
# trantor <clucko3@gmail.com>, 2014
# tbull <tbull@fedoraproject.org>, 2009
# Emilio Salano <unkn0wn_user@posteo.de>, 2016
# Ettore Atalan <atalanttore@googlemail.com>, 2015,2017
# gerhard <listmember@rinnberger.de>, 2013
# jugendhacker <julian.ribbeck@gmx.de>, 2017
# Konstantin BB, 2015
# Marcus Nitzschke <marcusni@gmx.de>, 2008
# Mario Baier <mario.baier26@gmx.de>, 2013
# matsa <matsa@riseup.net>, 2012
# Maximilian Schuster <maxschu0@outlook.com>, 2015
# max weber, 2015,2018
# M H <phi@posteo.de>, 2013
# Mike <inactive+webratte@transifex.com>, 2015
# mo <moritz@torservers.net>, 2013
# noble <noble@posteo.de>, 2013
# Chris <ppt23@lkj.hopto.org>, 2012
# Sacro <Scion@T-Online.de>, 2013
# Sacro <Scion@T-Online.de>, 2012
# Sebastian Fiele <rawsta@rawsta.de>, 2017
# spriver, 2018
# spriver, 2015
# stefanf <inactive+stefanf@transifex.com>, 2015
# Tobias Bannert <tobannert@gmail.com>, 2013
# Tobias Bannert <tobannert@gmail.com>, 2013-2014
# try once, 2015
# rike, 2014-2015,2017
# Victoria Kerley <VKerley75@gmail.com>, 2013
# Wolf <vv01f@riseup.net>, 2017
msgid ""
msgstr ""
"Project-Id-Version: The Tor Project\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-08-16 11:40+0200\n"
"PO-Revision-Date: 2018-08-16 14:46+0000\n"
"Last-Translator: carolyn <carolyn@anhalt.org>\n"
"Language-Team: German (http://www.transifex.com/otf/torproject/language/de/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: de\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: ../tails_installer/creator.py:100
msgid "You must run this application as root"
msgstr "Sie müssen diese Anwendung mit Administrationsrechten ausführen"

#: ../tails_installer/creator.py:146
msgid "Extracting live image to the target device..."
msgstr "Live-Abbild auf das Zielmedium extrahieren …"

#: ../tails_installer/creator.py:153
#, python-format
msgid "Wrote to device at %(speed)d MB/sec"
msgstr "Es wurde mit %(speed)d MB/sek auf das Gerät geschrieben"

#: ../tails_installer/creator.py:296
#, python-format
msgid ""
"There was a problem executing the following command: `%(command)s`.\n"
"A more detailed error log has been written to '%(filename)s'."
msgstr "Es gab ein Problem beim Ausführen des folgenden Befehls »%(command)s«. \nEine detailreichere Fehlerbeschreibung ist hier zu finden: »%(filename)s«"

#: ../tails_installer/creator.py:315
msgid "Verifying SHA1 checksum of LiveCD image..."
msgstr "SHA1-Prüfsumme des LiveCD-Abbilds wird überprüft …"

#: ../tails_installer/creator.py:319
msgid "Verifying SHA256 checksum of LiveCD image..."
msgstr "SHA256-Prüfsumme des LiveCD-Abbilds wird überprüft …"

#: ../tails_installer/creator.py:335
msgid ""
"Error: The SHA1 of your Live CD is invalid.  You can run this program with "
"the --noverify argument to bypass this verification check."
msgstr "Fehler: Die SHA1-Prüfsumme Ihrer Live-CD ist ungültig. Sie können dieses Programm mit dem '--noverify'-Argument ausführen, um die Verifikationsprüfung zu umgehen."

#: ../tails_installer/creator.py:341
msgid "Unknown ISO, skipping checksum verification"
msgstr "Unbekanntes ISO-Abbild, überspringe Prüfsummenüberprüfung"

#: ../tails_installer/creator.py:353
#, python-format
msgid ""
"Not enough free space on device.\n"
"%dMB ISO + %dMB overlay > %dMB free space"
msgstr "Nicht genug Speicherplatz auf dem Medium.\n%dMB ISO + %dMB Überlagerung > %dMB freier Speicher"

#: ../tails_installer/creator.py:360
#, python-format
msgid "Creating %sMB persistent overlay"
msgstr "%s MB beständige Speicherpartition wird erstellt"

#: ../tails_installer/creator.py:421
#, python-format
msgid "Unable to copy %(infile)s to %(outfile)s: %(message)s"
msgstr "%(infile)s konnten nicht nach %(outfile)s kopiert werden: %(message)s"

#: ../tails_installer/creator.py:435
msgid "Removing existing Live OS"
msgstr "Entferne bestehendes Live-OS"

#: ../tails_installer/creator.py:444 ../tails_installer/creator.py:456
#, python-format
msgid "Unable to chmod %(file)s: %(message)s"
msgstr "Berechtigungen von %(file)s konnten nicht geändert werden: %(message)s"

#: ../tails_installer/creator.py:449
#, python-format
msgid "Unable to remove file from previous LiveOS: %(message)s"
msgstr "Datei einer älteren Live-System-Version konnte nicht entfernt werden: %(message)s"

#: ../tails_installer/creator.py:462
#, python-format
msgid "Unable to remove directory from previous LiveOS: %(message)s"
msgstr "Ordner einer älteren Live-System-Version konnte nicht entfernt werden: %(message)s"

#: ../tails_installer/creator.py:510
#, python-format
msgid "Cannot find device %s"
msgstr "Kann Gerät %s nicht finden"

#: ../tails_installer/creator.py:711
#, python-format
msgid "Unable to write on %(device)s, skipping."
msgstr "Auf %(device)s kann nicht geschrieben werden. Überspringe."

#: ../tails_installer/creator.py:741
#, python-format
msgid ""
"Some partitions of the target device %(device)s are mounted. They will be "
"unmounted before starting the installation process."
msgstr "Einige Partitionen des Ziellaufwerks %(device)s sind eingehängt. Sie werden ausgehängt, bevor die Installation beginnt."

#: ../tails_installer/creator.py:784 ../tails_installer/creator.py:1008
msgid "Unknown filesystem.  Your device may need to be reformatted."
msgstr "Unbekanntes Dateisystem. Eventuell muss das Speichermedium neu formatiert werden."

#: ../tails_installer/creator.py:787 ../tails_installer/creator.py:1011
#, python-format
msgid "Unsupported filesystem: %s"
msgstr "Nicht unterstütztes Dateisystem: %s"

#: ../tails_installer/creator.py:805
#, python-format
msgid "Unknown GLib exception while trying to mount device: %(message)s"
msgstr "Unbekannte GLib-Ausnahme beim Versuch das Gerät einzuhängen: %(message)s"

#: ../tails_installer/creator.py:810
#, python-format
msgid "Unable to mount device: %(message)s"
msgstr "Speichermedium konnte nicht eingehängt werden: %(message)s"

#: ../tails_installer/creator.py:815
msgid "No mount points found"
msgstr "Keine Einhängepunkte gefunden"

#: ../tails_installer/creator.py:826
#, python-format
msgid "Entering unmount_device for '%(device)s'"
msgstr "unmount_device für »%(device)s«"

#: ../tails_installer/creator.py:836
#, python-format
msgid "Unmounting mounted filesystems on '%(device)s'"
msgstr "Eingehängtes Dateisystem auf %(device)s wird ausgehängt"

#: ../tails_installer/creator.py:840
#, python-format
msgid "Unmounting '%(udi)s' on '%(device)s'"
msgstr "'%(udi)s' auf '%(device)s' wird ausgehängt"

#: ../tails_installer/creator.py:851
#, python-format
msgid "Mount %s exists after unmounting"
msgstr "Mount %s existiert noch nach dem Aushängen"

#: ../tails_installer/creator.py:864
#, python-format
msgid "Partitioning device %(device)s"
msgstr "Speichermedium %(device)s wird partitioniert"

#: ../tails_installer/creator.py:993
#, python-format
msgid "Unsupported device '%(device)s', please report a bug."
msgstr "Nicht unterstütztes Medium »%(device)s«, bitte melden Sie einen Fehler."

#: ../tails_installer/creator.py:996
msgid "Trying to continue anyway."
msgstr "Es wird trotzdem versucht, fortzufahren."

#: ../tails_installer/creator.py:1005 ../tails_installer/creator.py:1401
msgid "Verifying filesystem..."
msgstr "Dateisystem wird überprüft …"

#: ../tails_installer/creator.py:1029
#, python-format
msgid "Unable to change volume label: %(message)s"
msgstr "Datenträgerbeschreibung konnte nicht geändert werden: %(message)s"

#: ../tails_installer/creator.py:1034 ../tails_installer/creator.py:1434
msgid "Installing bootloader..."
msgstr "Bootloader wird installiert …"

#: ../tails_installer/creator.py:1061
#, python-format
msgid "Could not find the '%s' COM32 module"
msgstr "COM32-Modul '%s' konnte nicht gefunden werden"

#: ../tails_installer/creator.py:1069 ../tails_installer/creator.py:1452
#, python-format
msgid "Removing %(file)s"
msgstr "Entferne %(file)s"

#: ../tails_installer/creator.py:1183
#, python-format
msgid "%s already bootable"
msgstr "%s bereits bootfähig"

#: ../tails_installer/creator.py:1203
msgid "Unable to find partition"
msgstr "Partition konnte nicht gefunden werden"

#: ../tails_installer/creator.py:1226
#, python-format
msgid "Formatting %(device)s as FAT32"
msgstr "%(device)s wird als FAT32 formatiert"

#: ../tails_installer/creator.py:1286
msgid "Could not find syslinux' gptmbr.bin"
msgstr "gptmbr.bin von syslinux konne nicht gefunden werden"

#: ../tails_installer/creator.py:1299
#, python-format
msgid "Reading extracted MBR from %s"
msgstr "Extrahierter MBR von %s wird gelesen"

#: ../tails_installer/creator.py:1303
#, python-format
msgid "Could not read the extracted MBR from %(path)s"
msgstr "Extrahierter MBR aus %(path)s konnte nicht gelesen werden"

#: ../tails_installer/creator.py:1316 ../tails_installer/creator.py:1317
#, python-format
msgid "Resetting Master Boot Record of %s"
msgstr "Master-Boot-Record von %s wird zurückgesetzt"

#: ../tails_installer/creator.py:1322
msgid "Drive is a loopback, skipping MBR reset"
msgstr "Das Laufwerk ist ein Loopback, MBR zurücksetzen wird übersprungen"

#: ../tails_installer/creator.py:1326 ../tails_installer/creator.py:1580
#, python-format
msgid "Calculating the SHA1 of %s"
msgstr "SHA-1-Summe von %s wird errechnet"

#: ../tails_installer/creator.py:1351
msgid "Synchronizing data on disk..."
msgstr "Daten auf dem Datenträger werden synchronisiert…"

#: ../tails_installer/creator.py:1393
msgid "Error probing device"
msgstr "Fehler beim Prüfen des Speichermediums"

#: ../tails_installer/creator.py:1395
msgid "Unable to find any supported device"
msgstr "Es wurde kein unterstütztes Speichermedium gefunden"

#: ../tails_installer/creator.py:1405
msgid ""
"Make sure your USB key is plugged in and formatted with the FAT filesystem"
msgstr "Stellen Sie sicher, dass Ihr USB-Medium eingesteckt und mit einem FAT-Dateisystem formatiert ist"

#: ../tails_installer/creator.py:1408
#, python-format
msgid ""
"Unsupported filesystem: %s\n"
"Please backup and format your USB key with the FAT filesystem."
msgstr "Nicht unterstütztes Dateisystem: %s\nBitte erstellen Sie eine Datensicherung und formatieren Sie Ihren USB-Stick mit einem FAT-Dateisystem."

#: ../tails_installer/creator.py:1475
msgid ""
"Unable to get Win32_LogicalDisk; win32com query did not return any results"
msgstr "Win32_LogicalDisk konnte nicht ermittelt werden; der win32com-Aufruf lieferte keinerlei Resultate"

#: ../tails_installer/creator.py:1528
msgid "Cannot find"
msgstr "Kann nicht gefunden werden"

#: ../tails_installer/creator.py:1529
msgid ""
"Make sure to extract the entire tails-installer zip file before running this"
" program."
msgstr "Stellen Sie sicher, dass die gesamte ZIP-Datei des tails-installer extrahiert wurde, bevor Sie dieses Programm ausführen."

#: ../tails_installer/gui.py:69
#, python-format
msgid "Unknown release: %s"
msgstr "Unbekannte Version: %s"

#: ../tails_installer/gui.py:73
#, python-format
msgid "Downloading %s..."
msgstr "%s wird heruntergeladen …"

#: ../tails_installer/gui.py:213
msgid ""
"Error: Cannot set the label or obtain the UUID of your device.  Unable to "
"continue."
msgstr "Fehler: Die Bezeichnung kann nicht gesetzt oder die UUID Ihres Speichermediums nicht ermittelt werden. Fortfahren nicht möglich."

#: ../tails_installer/gui.py:260
#, python-format
msgid "Installation complete! (%s)"
msgstr "Installation abgeschlossen! (%s) "

#: ../tails_installer/gui.py:265
msgid "Tails installation failed!"
msgstr "Tails-Installation fehlgeschlagen!"

#: ../tails_installer/gui.py:361
msgid ""
"Warning: This tool needs to be run as an Administrator. To do this, right "
"click on the icon and open the Properties. Under the Compatibility tab, "
"check the \"Run this program as an administrator\" box."
msgstr "Warnung: Dieses Werkzeug muss mit Administrationsrechten ausgeführt werden. Um dies zu tun, klicken Sie mit der rechten Maustaste auf das Programmsymbol und öffnen Sie dort die Eigenschaften. Setzen Sie nun einen Haken bei »Programm als Administrator ausführen« im Reiter »Kompatibilität«."

#: ../tails_installer/gui.py:373
msgid "Tails Installer"
msgstr "Tails-Installationsprogramm"

#: ../tails_installer/gui.py:423 ../data/tails-installer.ui.h:2
msgid "Clone the current Tails"
msgstr "Aktuelles Tails klonen"

#: ../tails_installer/gui.py:430 ../data/tails-installer.ui.h:3
msgid "Use a downloaded Tails ISO image"
msgstr "Heruntergeladenes Tails-ISO-Image verwenden"

#: ../tails_installer/gui.py:467
msgid "Upgrade"
msgstr "Aktualisieren"

#: ../tails_installer/gui.py:469
msgid "Manual Upgrade Instructions"
msgstr ""

#: ../tails_installer/gui.py:471
msgid "https://tails.boum.org/upgrade/"
msgstr ""

#: ../tails_installer/gui.py:479 ../data/tails-installer.ui.h:7
msgid "Install"
msgstr "Installieren"

#: ../tails_installer/gui.py:482 ../data/tails-installer.ui.h:1
msgid "Installation Instructions"
msgstr "Installationsanweisungen"

#: ../tails_installer/gui.py:484
msgid "https://tails.boum.org/install/"
msgstr ""

#: ../tails_installer/gui.py:490
#, python-format
msgid "%(size)s %(vendor)s %(model)s device (%(device)s)"
msgstr "%(size)s %(vendor)s %(model)s Gerät (%(device)s)"

#: ../tails_installer/gui.py:502
msgid "No ISO image selected"
msgstr "Kein ISO-Abbild ausgewählt"

#: ../tails_installer/gui.py:503
msgid "Please select a Tails ISO image."
msgstr "Bitte wählen Sie ein Tails-ISO-Abbild aus."

#: ../tails_installer/gui.py:545
msgid "No device suitable to install Tails could be found"
msgstr "Es konnte kein passendes Gerät zur Installation von Tails gefunden werden"

#: ../tails_installer/gui.py:547
#, python-format
msgid "Please plug a USB flash drive or SD card of at least %0.1f GB."
msgstr "Bitte stecken Sie ein USB-Flash-Laufwerk oder eine SD-Karte mit mindestens %0.1f GB ein."

#: ../tails_installer/gui.py:581
#, python-format
msgid ""
"The USB stick \"%(pretty_name)s\" is configured as non-removable by its "
"manufacturer and Tails will fail to start on it. Please try installing on a "
"different model."
msgstr "Der USB-Stick \"%(pretty_name)s\" wurde vom Hersteller als nicht entfernbar konfiguriert und somit würde Tails nicht davon starten. Bitte versuchen Sie es auf einem anderen Moddel zu installieren."

#: ../tails_installer/gui.py:591
#, python-format
msgid ""
"The device \"%(pretty_name)s\" is too small to install Tails (at least "
"%(size)s GB is required)."
msgstr "Der Datenträger \"%(pretty_name)s\" ist zu klein, um Tails darauf zu installieren (mindestens %(size)s GB werden benötigt)."

#: ../tails_installer/gui.py:604
#, python-format
msgid ""
"To upgrade device \"%(pretty_name)s\" from this Tails, you need to use a downloaded Tails ISO image:\n"
"https://tails.boum.org/install/download"
msgstr "Um das Gerät \"%(pretty_name)s\" von diesem Tails zu aktualisieren, müssen Sie ein heruntergeladenes Tails ISO Abbild benutzen:\nhttps://tails.boum.org/install/download"

#: ../tails_installer/gui.py:625
msgid "An error happened while installing Tails"
msgstr "Bei der Installation von Tails ist ein Fehler aufgetreten"

#: ../tails_installer/gui.py:637
msgid "Refreshing releases..."
msgstr "Veröffentlichungen werden aktualisiert …"

#: ../tails_installer/gui.py:642
msgid "Releases updated!"
msgstr "Veröffentlichungen wurden aktualisiert!"

#: ../tails_installer/gui.py:695
msgid "Installation complete!"
msgstr "Installation abgeschlossen!"

#: ../tails_installer/gui.py:744
msgid "Unable to mount device"
msgstr "Speichermedium kann nicht einhängt werden"

#: ../tails_installer/gui.py:751 ../tails_installer/gui.py:783
msgid "Confirm the target USB stick"
msgstr "Bestätigen Sie den Ziel-USB-Stick"

#: ../tails_installer/gui.py:752
#, python-format
msgid ""
"%(size)s %(vendor)s %(model)s device (%(device)s)\n"
"\n"
"All data on this USB stick will be lost."
msgstr "%(size)s %(vendor)s %(model)s Gerät (%(device)s)\n\nAlle Daten auf diesem USB-Stick gehen verloren."

#: ../tails_installer/gui.py:770
#, python-format
msgid "%(parent_size)s %(vendor)s %(model)s device (%(device)s)"
msgstr "%(parent_size)s %(vendor)s %(model)s Gerät (%(device)s)"

#: ../tails_installer/gui.py:778
msgid ""
"\n"
"\n"
"The persistent storage on this USB stick will be preserved."
msgstr "\n\nDer beständige Speicher auf diesem USB-Stick bleibt erhalten."

#: ../tails_installer/gui.py:779
#, python-format
msgid "%(description)s%(persistence_message)s"
msgstr "%(description)s%(persistence_message)s"

#: ../tails_installer/gui.py:821
msgid "Download complete!"
msgstr "Herunterladen abgeschlossen!"

#: ../tails_installer/gui.py:825
msgid "Download failed: "
msgstr "Herunterladen fehlgeschlagen:"

#: ../tails_installer/gui.py:826
msgid "You can try again to resume your download"
msgstr "Sie können versuchen, das Herunterladen fortzusetzen"

#: ../tails_installer/gui.py:834
msgid ""
"The selected file is unreadable. Please fix its permissions or select "
"another file."
msgstr "Die ausgewählte Datei konnte nicht gelesen werden. Bitte korrigieren Sie die Berechtigungen oder wählen Sie eine andere Datei aus."

#: ../tails_installer/gui.py:840
msgid ""
"Unable to use the selected file.  You may have better luck if you move your "
"ISO to the root of your drive (ie: C:\\)"
msgstr "Die ausgewählte Datei kann nicht verwendet werden. Vielleicht haben Sie mehr Glück, wenn Sie das ISO-Abbild in das Wurzelverzeichnis Ihres Laufwerkes verschieben."

#: ../tails_installer/gui.py:846
#, python-format
msgid "%(filename)s selected"
msgstr "%(filename)s ausgewählt"

#: ../tails_installer/source.py:28
msgid "Unable to find LiveOS on ISO"
msgstr "Kein LiveOS auf der ISO gefunden"

#: ../tails_installer/source.py:34
#, python-format
msgid "Could not guess underlying block device: %s"
msgstr "Das betroffene Blockgerät konnte nicht festgestellt werden: %s"

#: ../tails_installer/source.py:49
#, python-format
msgid ""
"There was a problem executing `%s`.\n"
"%s\n"
"%s"
msgstr "Es gab ein Problem bei der Ausführung von `%s`.\n%s\n%s"

#: ../tails_installer/source.py:63
#, python-format
msgid "'%s' does not exist"
msgstr "'%s' ist nicht vorhanden"

#: ../tails_installer/source.py:65
#, python-format
msgid "'%s' is not a directory"
msgstr "'%s' ist kein Ordner"

#: ../tails_installer/source.py:75
#, python-format
msgid "Skipping '%(filename)s'"
msgstr "'%(filename)s' wird übersprungen"

#: ../tails_installer/utils.py:44
#, python-format
msgid ""
"There was a problem executing `%s`.%s\n"
"%s"
msgstr "Es gab ein Problem bei der Ausführung von `%s`.%s\n%s"

#: ../tails_installer/utils.py:124
msgid "Could not open device for writing."
msgstr "Gerät konnte nicht zum Schreiben geöffnet werden."

#: ../data/tails-installer.ui.h:4
msgid "Select a distribution to download:"
msgstr "Wählen Sie eine Distribution zum Herunterladen aus:"

#: ../data/tails-installer.ui.h:5
msgid "Target USB stick:"
msgstr "Ziel-USB-Stick:"

#: ../data/tails-installer.ui.h:6
msgid "Reinstall (delete all data)"
msgstr "Neu installieren (alle Daten löschen)"
